junit5-extension/src/main/java/io/kroxylicious/testing/kafka/clients/CloseableConsumer.java

```diff
@@ -26,7 +26,22 @@
 import org.apache.kafka.common.PartitionInfo;
 import org.apache.kafka.common.TopicPartition;
 
+/**
+ * The type Closeable consumer.
```

Comment:
```suggestion
 * Provides a simple wrapper around a Kafka Consumer to redirect `close()` so that it has a sensible timeout and can thus be safely used in a try-with-resources block. All other methods delegate to the wrapped Consumer client. 
```